src/injection/react-native-web.js

```diff
@@ -10,7 +10,7 @@ const {
   Dimensions,
   Easing,
 } = require('react-native-web');
-const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry');
+const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry').default;
```

Comment:
@lelandrichardson any reason you use require over import statements?

@necolas It could be nice to have resolve in StyleSheet too :)